### ISSUE TITLE
Review backup auto delete on staging

### ISSUE THREAD

**Issue by georgemillard** [emfoundation/server-maintenance#37]
We have 33GB of backups in `staging.ellenmacarthurfoundation.org` on our staging server.

![Screen Shot 2020-02-10 at 11 30 10](https://user-images.githubusercontent.com/25058990/74150008-fd345e00-4c00-11ea-8d5f-9df222452500.png)

I'll clean them out. @lettie16 could you look into your scripts and the auto delete function?

Also please review the state of things on S3, as they seem to not be backing up there properly as well...

- [ ] Craft runtime backups need automated pruning
- [x] Scripts are fine, but S3 buckets are keeping to many

Maybe keep 2 months max.

**Comment by georgemillard** [comment]
@lettie16 There are also 9G of backups for the main emf site, about 5 months, which feels like more than we need to keep. Could you check these as well, please?
Thanks :-)

**Comment by lettie16** [comment]
Cron job tested on staging and now set up to remove files older than 30 days every Friday at 4.20 in the morning. So diarised for Monday 17th.
Backup files - Jan 13, Jan 14 & Jan 15 should be removed if cron ran ok. Once we have checked for a couple of weeks. I will implement it on the live server.

**Comment by georgemillard** [comment]
Sounds good 👍

**Comment by lettie16** [comment]
Lifecycle events are now set up on S3 to remove db files older than 60 days. Not sure when S3 runs these events so will check tomorrow to see if it has worked.

**Comment by lettie16** [comment]
S3 all working. Now removes files from:
emf/ellenmacarthurfoundation/db
emf/ellenmacarthurfoundation/craft/backups

Not set up on staging as we don't back staging up to S3.
Not set up on DIF as this is being retired
One set up already on NPEC

**Comment by georgemillard** [comment]
Great, thanks @lettie16.
Can you check in on this in say a couple of weeks to see it is still working as expected, then close this issue?

**Comment by lettie16** [comment]
Yep. Still need to check the server cron on staging first and then implement it on live. So will be open for a few weeks yet.

**Comment by georgemillard** [comment]
👍
